integration-tests/scripts/build-zeitgeist-spec.sh

```
@@ -2,10 +2,68 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 # Exit on any error
-set -e
+set -euo pipefail
 
 # Always run the commands from the "integration-tests" dir
 cd $(dirname $0)/..
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)